docs: import in testing (#1544)
posva/pinia: packages/docs/cookbook/testing.md

```js
import { mount } from '@vue/test-utils'
import { createTestingPinia } from '@pinia/testing'

> import { useSomeStore } from '@/stores/some.js' // or wherever else you defined the store

Reviewer: ```suggestion
// import any store you want to interact with in tests
import { useSomeStore } from '@/stores/myStore'
```
